### DISCUSSION TITLE
[Question] conversation starter

### DISCUSSION THREAD

**Discussion by avradip** [danny-avila/LibreChat#4370] [Q&A] [answered]
If i want to put some sample suggested questions on the landing page, is that possible to do right now? I saw there is a ConvoStarter component in the client code, but not sure whether it implements the feature that I am looking for.

New to this project, looks amazing so far.

**Comment by danny-avila** [comment, marked as answer]
Right now this is only available for Assistants, but will then come to our own Agents system, followed by regular Endpoints
